src/components/WorkStyle/styles.css

```diff
@@ -0,0 +1,35 @@
+.work-style::before {
+  content: '';
+  background: no-repeat url('./img/work-style/dots.png');
+  background-size: auto;
+  background-position-x: 5%;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  z-index: -1;
+}
+
+.work-style__items {
+  width: 95%;
+}
+
+.showcase-row__heading {
+  padding-left: 15%;
```

Comment:
I suggest increasing horizontal padding left and right to force the title to be in two lines and the alignment issue will be solved

Reply:
Thanks for your suggestion.